lms/templates/student_account/account_settings_section.underscore

```diff
@@ -0,0 +1,18 @@
+<% _.each(sections, function(section) { %>
+    <div class="section">
+        <% if (section.subtitle) { %>
+            <h4 class="account-settings-header-subtitle"><%- gettext(section.subtitle) %></h4>
```

Comment:
On these two tabs, the value of subtitle is much too long to be considered a proper heading:
"These settings include basic information about your account. You can also specify additional information and see your linked social accounts on this page."
and
"You can link your social media accounts to your edX account to make signing in to edx.org and the edX mobile apps easier."

These should just be regular paragraphs.  Ideally, they would include IDrefs that could be referenced by aria-describedby on the Tabs ("About" and "Linked accounts") since they describe the tabs.